catalogue/aarch64/tests/LB+rel+CAS-ok-RsM-addr.litmus

```diff
@@ -0,0 +1,23 @@
+AArch64 LB+rel+CAS-ok-RsM-addr
+
+{
+  x = 0;
+  y = 0;
+  z = 0;
+
+  0: X0 = x;      1: X0 = x;
+  0: X1 = y;      1: X1 = y;
+                  1: X2 = z;
+}
+
+P0            | P1                      ;
+
+LDR W3, [X0]  | LDR W3, [X1]            ;
+MOV W4, #1    | EOR W4, W3, W3          ;
+STLR W4, [X1] | CAS W4, W5, [X2]        ;
```

Comment:
What is the dependency you expect here (between the LDR and the CAS)? Because currently this is not a data.